app/src/androidTest/java/ch/epfl/qedit/quiz/question/QuestionFragmentTest.java

```diff
@@ -29,6 +30,9 @@
 
     @Before
     public void setup() {
+
+        Bundle bundle = new Bundle();
```

Comment:
I think we should delete this line, right ?

Reply:
Yes